Automattic/jetpack
# add support to delete connection owner via CLI

#### Changes proposed in this Pull Request:

* Adds support to delete connection owner via jetpack disconnect CLI command

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Set up a full connection
* run `wp jetpack disconnect user 1` (where `1` is your admin/connection owner user id)
* Make sure you get a error message with the user login and email
* Make sure the error message knows it is the connection owner and suggests you to use `--force`
* run `wp jetpack disconnect user 1 --force`
* Make sure it succeeds
* Visit your dashboard and confirm you're now in site-only connection mode
* try to delete the user again and make sure it fails and the error message says the user is not connected
